examples/multiple-clusters-with-allocation-endpoint/terraform/modules/gke_with_agones/main.tf

```diff
@@ -0,0 +1,113 @@
+# Copyright 2022 Google LLC All Rights Reserved.
```

Comment:
Do we need to replicate this Terraform stuff we have in https://github.com/googleforgames/agones/tree/main/install/terraform/modules (this is duplicate right, I'm not missing something?)

Could we reference it relatively, or do it similarly to how we do things here:

https://github.com/googleforgames/agones/blob/6e57d34ed1c0250555b986ff83c7f9745ac11f8b/examples/terraform-submodules/gke/module.tf#L82-L100

Reply:
In short:
It is not 1:1 but we could use it. I would need to update stuff in https://github.com/googleforgames/agones/tree/main/install/terraform/modules  to make it work in the same way it currently works and in "my way" where it it needed. 

Bit longer explenation:
In the end it was "by choice" I wanted to avoid making any updates to /install/terraform/modules but those updates are nothing major so I guess I can do it but I will need to modify existing scripts in /install/terraform... a bit...

For GKE i use dedicate service account instead of default one. 
Some additional parameters in helm3 module agones.allocator.* are easy ones agones.metrics.* managable. also managable bit more puzzled with what to do with agones.serviceaccount.controller.* but I'll think of something. 

So do you think I should do it ?

Reply:
Yes, I do. if we are providing examples, we should eat our own dogfood. It's only going to confuse things if we have competing solutions.

If there are good reasons to expand the feature set of the existing solutions, and there is no downside, we should add them and make the developer experience as smooth as possible.